help/develop/how-to/report-tickets.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Report bugs and feature requests"
+---
+
+This guide provides an easy set of rules for reporting issues and managing
+tickets in Datagrok. We use two issue tracking systems:
+
+* [JIRA](https://reddata.atlassian.net/)
+for internal issues integrated with BitBucket
+* [GitHub Tracker](https://github.com/datagrok-ai/public/issues)
+for all externally reported issues.
+
+We recommend reporting issues in JIRA if you have access to it, but if you
+don't, use GitHub Tracker. The rules of the bug report format adopted by the
+company do not differ in both issue tracking systems.
+
+## Reporting an issue
+
+A good ticket should have a type, title (summary), description, assignee,
+priority, and due date. Some tickets should be included in the projects or
+epics. Here we provide recommendations for each item:
+
+* **Issue type**. In GitHub, we use two labels to type an issue: _bug_ and
+  _enhancement_. In Jira, we use similar issue types: _bug_, _improvement_
+  (enhancement of existing functionality or UI), and _new feature_ (requesting
+  new capability or software feature).
+* **Title (Summary)**. Start the title with the platform module or package name,
+  then name unexpected behavior for a bug and a new feature name or improvement
+  summary for an enhancement. If relevant, use a hierarchical representation of
+  the issue location: list modules and submodules in their logical order using
+  the “|” symbol.
+
+|Issue type|Formula|Example|
+|----------|-------|-------|
+|Bug report| [Platform module] or [Package name] : [Unexpected behavior]| Line chart: Y-axis label is broken|
+|New feature or Improvement| [Platform module] or [Package name] : [New feature name] or [Improvement summary]| Bar Chart: context menu harmonization|
+
+* **Description**. Is important for testing and writing release notes. For an
```

Comment:
Do not start sentences with "is". But continuing the sentence like that is ok:

**Description** is important